pkgs/development/python-modules/trezor/default.nix

```diff
@@ -39,12 +42,17 @@ buildPythonPackage rec {
     ecdsa
     libusb1
     mnemonic
+    noiseprotocol
+    pillow
     requests
     shamir-mnemonic
     slip10
     typing-extensions
   ]
-  ++ lib.optionals stdenv.hostPlatform.isLinux [ trezor-udev-rules ];
+  ++ lib.optionals stdenv.hostPlatform.isLinux [
+    bleak
```

Comment:
That belongs in `optional-dependencies.ble`.

Reply:
These are needed to support the new Trezor Safe 7 model. I prefer to have this functionality in nixpkgs by default.